### ISSUE TITLE
Selected state of `Segmented Control` not A11Y compliant with hover state

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#4594]
### JS Framework

all

### Porsche Design System version

4.4.0

### On which device(s) you can see the bug?

all

### In which browsers you can see the bug?

all

### Browser version

latest

### Reproduction Example

Storefront

### Steps to reproduce the behavior

1. Go to https://designsystem.porsche.com/v4/components/segmented-control/configurator/
2. Hover over a `Segmented Control` item and click it
3. See error: no visual distinguishable of those two states

### Expected behavior

The **selected** state should be visually distinguishable from the **hover** state. Additionally improve the **selected** state to be more distinguishable from all other states so that this state stands really out.

### Relevant log output

```shell

```
